.travis.yml

```diff
@@ -2,8 +2,10 @@ sudo: false
 language: python
 python:
 - '2.7'
+- '3.6'
 script: 
 - "python setup.py install"
+script: "pip install -e ."
```

Comment:
This doesn't test installation step. If you want tot test you should:

1. Make the package `python setup.py sdist`
2. Install this package from `dist` folder

Reply:
Done!